lib/utils/hierarchy_dataframe.py

```diff
@@ -21,6 +21,7 @@ def hierarchy_dataframe(metadata: pd.DataFrame, hierarchy_columns: list) -> pd.D
             sep='_',
             suffix='\w+') \
         .reset_index()[hierarchy_columns] \
+        .fillna('') \
```

Comment:
I am now questioning if this is desirable behavior or if we'd prefer no side effects of hidden type casting.